src/styles/styles.js

```diff
@@ -2438,7 +2436,7 @@ const styles = {
 
     hiddenElementOutsideOfWindow: {
         position: 'absolute',
-        top: 0,
+        top: -10000,
```

Comment:
Hmm, why is this changing? Since opacity is 0 I would imagine this was already invisible?